tests/model_fields/test_filefield.py

```diff
@@ -72,6 +72,21 @@ def test_save_without_name(self):
             with self.assertRaisesMessage(SuspiciousFileOperation, msg):
                 document.save()
 
+    def test_save_content_file_without_name(self):
+        d = Document()
+        d.myfile = ContentFile(b"")
+        msg = "File for myfile must have the name attribute specified to be saved."
+        with self.assertRaisesMessage(FieldError, msg):
+            d.save()
```

Comment:
We can also test the note:

```suggestion
        with self.assertRaisesMessage(FieldError, msg) as cm:
            d.save()
            
        if PY311:
            self.assertEqual(
                cm.exception.__notes__,
                ["Pass a 'name' argument to ContentFile."]
            )
```